sql/functions/table_has_identity_columns.sql

```diff
@@ -0,0 +1,40 @@
+CREATE OR REPLACE FUNCTION table_has_identity_columns(
+    table_name text,
+    p_ignored_columns text[] DEFAULT NULL
+) RETURNS boolean AS $$
+DECLARE
+    identity_columns_count int;
```

Comment:
Appreciate this PR and will definitely look at getting it merged in when possible. Would you mind using the conventions I have in the other functions for parameter (prefix with `p_`)  and variable names (prefix with `v_`).